docs/packages/cookbook/article-tagging.rst

```diff
@@ -0,0 +1,93 @@
+Auto-Tagging and Querying Articles
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This example package collects plain text files and stores them in Steamship. It tags them by running a
+zero-shot classifier with a set of labels provided by the package user.  The results
+can then be queried by tag with a configurable threshold.
+
+..
+    TODO: Test this once oneai tagger is updated
+
+.. code-block:: python
+
+    """
+    This package accepts text documents, stores them, labels them with tags,
+    and retrieves them based on those tags.
+    """
+    from typing import Type, Dict, Any
+
+    from steamship import Block, File, Steamship, Tag
+    from steamship.invocable import Config, create_handler, post, PackageService
+
+
+    class ArticleTaggerPackage(PackageService):
+        class ArticleTaggerConfig(Config):
+            """Configuration required to instantiate this package."""
+
+            labels: str  # A comma-separated list of tags to apply to articles
+
+        def __init__(self, client: Steamship, config: Dict[str, Any] = None):
+            super().__init__(client, config)
+
+            # Instantiate a zero-shot classifier plugin
+            self.classifier_instance = client.use_plugin(
+                plugin_handle="zero-shot-tagger-default",
+                instance_handle="my-classifier",
+                config={"tag_kind": "tags",  # The tag.kind we want in the output
+                        "labels": self.config.labels,  # The labels we want to apply
+                        "multi_label": True  # multi-class classification
+                        }
+            )
+
+        # The config method allows your package to return a class
+        # that defines its required configuration.
+        # See <link> for more details.  This package doesn't have any specific
```

Comment:
does `<link>` get filled in via some other mechanism?

Reply:
No, I just didn't figure out how to do Sphinx relative links from inside a code block.  A quick googling leads me to believe you can't do that without css hackery, so I'm probably just going to use words for now.